common/app/views/mainAMP.scala.html

```diff
@@ -40,6 +42,22 @@
 
             @body
 
+            <div class="fc-container__inner">
+                <div class="fc-container__header__title">
+                    <span>popular</span>
+                </div>
+                <amp-list width="600" height="1000" layout="responsive" src="@if(Play.isDev) {https://www.thegulocal.com} else {@Configuration.ajax.url}/most-read-amp.json">
```

Comment:
It would work normally. In dev mode it just gives you a relative url. That wont work for amp-list though because it requires having a url that starts with `https`. Can you think of a better way of doing this for local development?